add: search/transactions (#2986)
dfinity/portal: docs/developer-docs/defi/rosetta/icp_rosetta/data_api/transactions.mdx

        }
    }
}
```

## SearchTransactions endpoint

> There exist multiple parameters that can be used to query for transactions. See the [full specification](https://docs.cdp.coinbase.com/rosetta/reference/searchtransactions/) to see what query parameters can be used. 

Reviewer: ```suggestion
There are several parameters that can be used to query transactions. See the [full specification](https://docs.cdp.coinbase.com/rosetta/reference/searchtransactions/) to see what query parameters can be used. 
```

> For example if you want to fetch all transactions in a given block range of maximal 10000 blocks where a specific
> account was involved in you can make a request to the `search/transactions` endpoint like the following:

Reviewer: ```suggestion
For example, if you want to fetch all transactions in a given block range (of maximum 10000 blocks) where a specific account was involved, you can make a request to the `search/transactions` endpoint:
```
